vnext/src-win/Libraries/Components/Pressable/Pressable.windows.js

```diff
@@ -309,6 +318,16 @@ function Pressable({
     hitSlop,
   };
 
+  const onPressWithFocus = React.useCallback(
+    (args: GestureResponderEvent) => {
+      if (focusable !== false && focusOnPress !== false) {
```

Comment:
`[supernit|non-blocking]` am I forgetting a JS basic for why this shouldn't be
```ts
if (!focusable && !focusOnPress)
```
?

Reply:
the difference is what the default value of undefined gets counted as.

Reply:
edit: oh because you have the default for this behavior to be active...